[2022-02-09 08:59:15,630] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-10-01T23:00:00+00:00 [queued]>
[2022-02-09 08:59:15,642] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-10-01T23:00:00+00:00 [queued]>
[2022-02-09 08:59:15,642] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:59:15,642] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 08:59:15,642] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:59:15,656] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2020-10-01 23:00:00+00:00
[2022-02-09 08:59:15,662] {standard_task_runner.py:52} INFO - Started process 217 to run task
[2022-02-09 08:59:15,669] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2020-10-01T23:00:00+00:00', '--job-id', '465', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingestee_gcs_dag.py', '--cfg-path', '/tmp/tmpkull7qyo', '--error-file', '/tmp/tmpjz990q53']
[2022-02-09 08:59:15,670] {standard_task_runner.py:77} INFO - Job 465: Subtask local_to_gcs_task
[2022-02-09 08:59:15,744] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-10-01T23:00:00+00:00 [running]> on host cc021fb126ed
[2022-02-09 08:59:15,795] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 08:59:15,820] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2020-10-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-10-01T23:00:00+00:00
[2022-02-09 08:59:23,830] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 08:59:23,844] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20201001T230000, start_date=20220209T085915, end_date=20220209T085923
[2022-02-09 08:59:23,867] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 08:59:23,907] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-09 09:14:45,047] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-10-01T23:00:00+00:00 [queued]>
[2022-02-09 09:14:45,063] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-10-01T23:00:00+00:00 [queued]>
[2022-02-09 09:14:45,064] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:14:45,064] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:14:45,064] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:14:45,078] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2020-10-01 23:00:00+00:00
[2022-02-09 09:14:45,086] {standard_task_runner.py:52} INFO - Started process 331 to run task
[2022-02-09 09:14:45,097] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2020-10-01T23:00:00+00:00', '--job-id', '563', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingesteee_gcs_dag.py', '--cfg-path', '/tmp/tmpw48rrd1o', '--error-file', '/tmp/tmpf0279dep']
[2022-02-09 09:14:45,098] {standard_task_runner.py:77} INFO - Job 563: Subtask local_to_gcs_task
[2022-02-09 09:14:45,183] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-10-01T23:00:00+00:00 [running]> on host 75953ff4f3d5
[2022-02-09 09:14:45,238] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:14:45,269] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2020-10-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-10-01T23:00:00+00:00
[2022-02-09 09:14:52,603] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 09:14:52,632] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20201001T230000, start_date=20220209T091445, end_date=20220209T091452
[2022-02-09 09:14:52,696] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:14:52,791] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
